src/common-types.ts

```diff
@@ -9,6 +9,7 @@ export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
 
 export interface Theme {
 	breakpoints: string[];
+	containers: string[];
```

Comment:
How about instead of adding it to the top-level theme, we add it to `container.maxWidth` similar to how each component is styled?